src/Pages/NotificationsSettings/KeywordsSettingForm.tsx

```diff
@@ -0,0 +1,141 @@
+import { CATEGORY_KEYWORD_OPTIONS, POSITION_KEYWORD_OPTIONS } from '@/Shared/study';
+import { KeywordForm } from '@/Components/Common/KeywordForm';
+import ChipMenu from '@/Components/Common/ChipMenu';
+import Button from '@/Components/Common/Button';
+import styled from 'styled-components';
+import { useForm } from 'react-hook-form';
+import { RecruitmentKeywordsForm } from '@/Types/notifications';
+import { useEditNotificationsKeywords } from '@/Hooks/notifications/useEditNotificationsKeywords';
+import { useState } from 'react';
+import { StackModal } from '@/Components/Modal/StackModal';
+import { Stack } from '@/Types/study';
+
+const handleValues = (selectedValues: number[], id: number) => {
+  return selectedValues.includes(id)
+    ? selectedValues.filter((selectedValue: number) => selectedValue != id)
```

Comment:
배열에 id 존재 여부에 따라 토글하는 로직을 잘 구현해주셨네요!
값을 비교할 때는 타입까지 체크하는 !== 를 사용하는게 좋을 것 같습니다 🙂
```suggestion
    ? selectedValues.filter((selectedValue: number) => selectedValue !== id)
```

Reply:
수정하였습니다👍